Add blogpost for week 15 (#1029)
packit/packit.dev: weekly/2025/week-15.md

---
title: Week 15 in Packit
date: 2025-04-14
authors: mmassari
tags:
  - 2025-April
  - 2025
  - April
---

## Week 15 (April 8th – April 14th)

> - We have implemented a set of options `osh_options` that allow you to customize OpenScanHub scans via config. ([packit#2570](https://github.com/packit/packit/pull/2570))

Reviewer: ```suggestion
- There is a new `osh_options` config option that allows you to customize OpenScanHub scans. ([packit#2570](https://github.com/packit/packit/pull/2570))
```

If this is documented already then we could also link the docs.
